<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QWidget" name="MainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="PushButton" name="tmpButton">
       <property name="text">
        <string>请选择模板文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LineEdit" name="lineEdit">
       <property name="placeholderText">
        <string>必填</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="PushButton" name="sofpButton">
       <property name="text">
        <string>请选择资产负载表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LineEdit" name="lineEdit_2">
       <property name="placeholderText">
        <string>可选</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="PushButton" name="profitButton">
       <property name="text">
        <string>请选择资利润表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LineEdit" name="lineEdit_3">
       <property name="placeholderText">
        <string>可选</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="PushButton" name="flowButton">
       <property name="text">
        <string>请选择流量表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LineEdit" name="lineEdit_4">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>可选</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="PrimaryPushButton" name="startButton">
     <property name="text">
      <string>开始生成</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PrimaryPushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
